website/docs/ecosystem/agentops.md

```diff
@@ -1,32 +1,80 @@
-# AgentOps
+# AgentOps 🖇️
 
-[!agentops-logo](img/ecosystem-agentops.png)
+<img src="https://raw.githubusercontent.com/AgentOps-AI/agentops/main/docs/img/banner.png" alt="AgentOps Logo" width="50%"/>
+
+[AgentOps](https://agentops.ai/?=autogen) provides session replays, metrics, and monitoring for agents.
+
+At a high level, AgentOps gives you the ability to monitor LLM calls, costs, latency, agent failures, multi-agent interactions, tool usage, session-wide statistics, and more. For more info, check out the [AgentOps Repo](https://github.com/AgentOps-AI/agentops).
+
+> Create an account and API key at [AgentOps.ai](https://agentops.ai/)
+
+<details open>
+  <summary>Agent Dashboard</summary>
+   <img src="https://raw.githubusercontent.com/AgentOps-AI/agentops/main/docs/img/overview.png" style="width: 90%;" alt="Agent Dashboard"/>
+</details>
+
+<details>
+  <summary>Session Statistics</summary>
+   <img src="https://raw.githubusercontent.com/AgentOps-AI/agentops/main/docs/img/session.png" style="width: 90%;" alt="Agent Dashboard"/>
+</details>
+
+<details>
+  <summary>Session Replay</summary>
+   <img src="https://raw.githubusercontent.com/AgentOps-AI/agentops/main/docs/img/replay.png" style="width: 90%;" alt="Agent Dashboard"/>
+</details>
+
+
+## Installation
 
-> Create an account at [AgentOps.ai](https://agentops.ai/)
 AgentOps works seamlessly with applications built using Autogen.
 
-Install AgentOps with
+1. **Install Autogen with AgentOps**
 ```bash
 pip install pyautogen[agentops]
```

Comment:
Again, not sure if we will be able to do this if autogen makes us remove agentops from setup.py